src/bentoml/_internal/marshal/dispatcher.py

```diff
@@ -118,7 +120,7 @@ def __init__(
         """
         self.max_latency_in_ms = max_latency_in_ms / 1000.0
         self.fallback = fallback
-        self.optimizer = Optimizer()
+        self.optimizer = Optimizer(self.max_latency_in_ms)
```

Comment:
should we rename this property from `max_latency_in_ms` to `max_latency_in_seconds`?

Reply:
Yes, we should. I'll make a quick change.

Reply:
I will make this change in my docs PR.

Reply:
@aarnphm why docs change? this should not be a user-facing change right?